_security-plugin/configuration/multi-auth.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: Multiple option authentication
+parent: Configuration
+nav_order: 3
+---
+
+# Multiple option authentication for Dashboards sign-in
+
+You can configure the sign-in window for OpenSearch Dashboards to provide either a single option for authenticating users at sign-in or multiple options. Currently, Dashboards supports basic authentication, OpenID Connect, and SAML as the multiple options.
+
+## General steps for configuring multiple option authentication
+
+1. Decide which types of authentication to make available at sign-in.
+1. Configure each authentication type, including an authentication domain for the identification provider (IdP) and the essential settings that give each type sign-in access to OpenSearch Dashboards. For OpenId Connect backend configuration, see [OpenID Connect]({{site.url}}{{site.baseurl}}/security-plugin/configuration/openid-connect/); For SAML backend configuration, see [SAML]({{site.url}}{{site.baseurl}}/security-plugin/configuration/saml/).
+1. Add, enable, and configure multiple option authentication settings in the `opensearch_dashboards.yml` file.
+
+## Enabling multiple option authentication
+
+By default, Dashboards provides basic authentication as a single option for signing in. To enable multiple options for authentication, begin by adding `opensearch_security.auth.multiple_auth_enabled` to the `opensearch_dashboards.yml` file and setting it to `true`.
```

Comment:
maybe keep parallel with all option descriptions in the section?
"begin by adding" --> "add `opensearch_security.auth.multiple_auth_enabled` to the `opensearch_dashboards.yml` file and set it to `true`."